src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -1,25 +1,17 @@
 ---
-title: "Dynamic Sampling Context (Experimental)"
+title: 'Dynamic Sampling Context'
 ---
 
-<Alert level="warning">
-
-This page is under active development.
-Specifications are not final and subject to change.
-Anything that sounds fishy probably is - nothing is set in stone.
-Opening PRs to improve this page is therefore highly encouraged!
-
-</Alert>
+All data sent to Sentry will end up in a [Trace](https://docs.sentry.io/concepts/key-terms/tracing/distributed-tracing/#traces-transactions-and-spans). Traces can be sampled by the <Link to="/sdk/performance/#tracessamplerate">`tracesSampleRate`</Link> or <Link to="/sdk/performance/#tracessampler">`tracesSampler`</Link> options in the SDKs.
 
-Traces sampling done through the <Link to="/sdk/performance/#tracessamplerate">`tracesSampleRate`</Link> or <Link to="/sdk/performance/#tracessampler">`tracesSampler`</Link> options in the SDKs has quite a few consquences for users of Sentry SDKs:
+Changing those options has quite a few consequences for users of Sentry SDKs:
 
 - Changing the sampling rate involved either redeploying applications (which is problematic in case of applications that are not updated automatically, i.e., mobile apps or physically distributed software) or building complex systems to dynamically fetch a sample rate.
 - The sampling strategy leverages head-based simple random sampling.
 - Employing sampling rules, for example, based on event parameters, is very complex. Sometimes this even requires users to have an in-depth understanding of the event schema.
 - While writing rules for singular **transactions** is possible, enforcing them on entire **traces** is infeasible.
 
-The solution for these problems is **Dynamic Sampling**.
-Dynamic Sampling allows users to configure **sampling rules** directly in the Sentry interface. Important: Sampling rules may be applied to **entire traces** or to a **single transaction**.
+**Dynamic Sampling** allows users to configure **sampling rules** directly in the Sentry interface. Those sampling rules may be applied to **entire traces** or to a **single transaction**. Dynamic Sampling allows Sentry to automatically adjust the amount of data retained based on the value of the data. For detailed information see [Dynamic Sampling](/dynamic-sampling/).
```

Comment:
Where can people configure DS in the UI?

Reply:
Yeah this is old, we should remove that information

Reply:
Ok, will remove

Reply:
Ok, will remove